src/sonic-device-data/tests/permitted_list

```diff
@@ -218,3 +218,11 @@ serdes_fec_enable
 pbmp_gport_stack
 reglist_enable
 scache_filename
+
+help_cli_enable
+ifp_inports_support_enable
+memlist_enable
```

Comment:
it already exits.

can you check duplicates?

Reply:
done!